arwen/host/execution_vmoutputs_test.go

```diff
@@ -345,7 +348,7 @@ func expectedVMOutputSameCtxRecursiveMutualMethods(_ []byte, recursiveCalls int)
 	)
 	account.Balance = big.NewInt(1000)
 	account.BalanceDelta = big.NewInt(0).Sub(big.NewInt(1), big.NewInt(1))
-	account.GasUsed = 25412
+	account.GasUsed = 67668
```

Comment:
why were these values changed ?

Reply:
In this specific case, it's because the same contract was calling itself with `executeOnSameContext()`, so it simply used more gas. The test currently instructs the contract to call itself 5 times, which uses 67668 gas units, but here are more values:

2 recursive calls: 25304 = 2 compilations + 2 executions
3 recursive calls: 37998 = 3 compilations + 3 executions
4 recursive calls: 50730 = 4 compilations + 4 executions
5 recursive calls: 67668 = 5 compilations + 5 executions
6 recursive calls: 84644 = 6 compilations + 6 executions

The previous value was computed differently.